hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -29,9 +29,68 @@ <h2 class="formH2">Complete your application</h2>
                         Get started
                     </a>
                 {% endif %}
+            {% elif application is not none and review is not none %}
+                <br />
+                {% if review.status == "Accepted" %}
```

Comment:
In general I think this section should only show if the application deadline has not passed. Even if they've accepted the RSVP, the page appears the same with only the status line having changed. 

We can probably do something like:
- If they've RSVPd yes and the deadline has passed, say thanks and see you there
- If they've RSVPd no and the deadline has passed, say we're sad but that they can't change it
- If they haven't RSVPd and the deadline has passed, say they missed the deadline